Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nikolaik/python-nodejs:python3.11-nodejs22-slim
```

Comment:
dumb question about docker. How is it we end up relying on what seems to be a random project or repo or not sure what it is? Who is this nikolaik guy?

Reply:
Is this introduce some hidden dependency on nodejs 22?